src/text-editor.coffee

```diff
@@ -2587,6 +2587,16 @@ class TextEditor extends Model
       maintainClipboard = true
     return
 
+
+  # Essential: For each selection, only copy hightlighted text. Line is coptied if no selected text and cursor is at the beginning of the line
```

Comment:
yeah, in an earlier version of this branch, that comment was true - but then based on some feedback from what I read in the discussion in #7103 I removed it, so it's no longer true. I'll remove that and update the rest